src/index.js

```diff
@@ -0,0 +1,64 @@
+import 'dotenv/config';
+import express from 'express';
+import userRouter from './routes/userRouter';
+import groupRouter from './routes/groupRouter';
+import authRouter from './routes/authRouter';
+import loginRouter from './routes/loginRouter';
+import logger from './util/logger';
+import { internalErr, notFound } from './util/constant';
+import { stringifyBody } from './util/util';
+import cors from 'cors';
+import * as HttpStatus from 'http-status-codes';
+
+const app = express();
+
+const domainsWhitelist = process.env.CORS_WHITELIST.split(',');
+const corsOptions = {
+    origin: (origin, callback) => {
+        if (domainsWhitelist.indexOf(origin) !== -1) {
+            callback(null, true);
+        } else {
+            callback(new Error(`origin: ${origin} isn't allowed by CORS`));
+        }
+    }
+};
+
+app.use(cors(corsOptions));
+
+app.use(express.json());
+
+app.use(['/users', '/groups'], authRouter);
```

Comment:
I meant smt like this:

`const securedRoutes = express.Router();
securedRoutes.use(authMiddleware);
securedRoutes.use('/users', userRouter);
securedRoutes.use('/groups', groupRouter);

app.use(securedRoutes);`

And also pay attention that authRouter it is not a router, but middleware. Rename it as `authMiddleware`